packages/java/parser-jvm-core/src/main/java/com/vaadin/fusion/parser/core/PluginsToolset.java

```diff
@@ -0,0 +1,52 @@
+package com.vaadin.fusion.parser.core;
+
+import java.util.Optional;
+import java.util.SortedSet;
+
+public class PluginsToolset {
+    private final SortedSet<Plugin> plugins;
+
+    public PluginsToolset(SortedSet<Plugin> plugins) {
+        this.plugins = plugins;
+    }
+
+    public Optional<Integer> comparePluginOrders(Plugin current,
```

Comment:
Do we really Optional wrapper? Could we just use `null` for `Optional.empty()`?

Reply:
`PluginsToolset` is supposed to be used by the developer who works on their own plugin. So I decided to make it clearer.

Reply:
ok